src/scss/buttons.scss

```diff
@@ -0,0 +1,17 @@
+@import './colors';
```

Comment:
Import colors only in `application.scss`. As you import all generics, those know the variables